backend/initial_setup.py

```diff
@@ -194,7 +195,8 @@ def __init__(self):
     @classmethod
     def get_version(cls):
         try:
-            # TODO use some sort of git lookup to get information
-            return "0.0.1"
-        except:
-            return "0.0.1"
+            return subprocess.check_output([
```

Comment:
Why not use a git python library?

Also does this work when run from different directories?

It may not be running from the git folder

Reply:
We know that anyone running this will have git installed, so I opted to not introduce another dependency to the service (in general I think it's better to minimize dependencies -- fewer places for a vulnerability, and fewer things to upgrade).

All git commands that don't refer to file paths can be run from anywhere in the repo, this one included (I just tested it in an inner directory to be sure).

Reply:
The first assumption is false.
I'm on Windows and do not have git command line installed.  We should make sure it won't crash on computers without git command line

Reply:
It won't crash, it will just return a default version of unknown. This seems like a reasonable addition to me